.ci/.npmrc-example

```diff
@@ -0,0 +1,4 @@
+scripts-prepend-node-path=true
+always-auth=true
+@yalesites-org:registry=https://npm.pkg.github.com/
```

Comment:
I understand why we need a .npmrc file in general, but why do we keep a separate one in the `.ci` directory? Should this just exist at the project root that way it is useful for local development too? Or is it best practice to not track this file?